rambo/app.py

```diff
@@ -323,7 +324,18 @@ def install_plugins(force=None, plugins=('all',)):
             if not force:
                 click.confirm('The plugin "%s" is not in our list of plugins. Attempt '
                           'to install anyway?' % plugin, abort=True)
-            _invoke_vagrant('plugin install %s' % plugin)
+            vagrant_general_command('plugin install %s' % plugin)
+
+def scp(ctx=None, locations=None):
+    assert(len(locations)==2)
```

Comment:
This is kinds of nasty. I'm assuming in the case the user would see an assertion that "len of locations not == 2". We know what the issue is, we could make that exception handling way nicer. Tell them how they probably screwed up and provide and example of what the input is supposed to look like.

Reply:
Accidentally added. Will revise.